.github/workflows/mit-libraries-ruby-ci.yml

```diff
@@ -0,0 +1,10 @@
+name: CI
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  shared:
+    uses: mitlibraries/.github/.github/workflows/ruby-shared-ci.yml@main
```

Comment:
It looks like this shared workflow is essentially the same as the `ci.yml` file that's being removed in this PR? I'm definitely okay with that.

Reply:
It's very similar but requires a normalized `*.lcov` filename. The default is the repo name but apps using the shared ci workflow all have to use the same filename so there is a related change in this repo to allow for moving to the share workflow in `test_helper.rb` as you noted in a comment on that file. So yes, those two changes together make the shared workflow, er, work :)